docs/source/using/managing-minishift.adoc

```diff
@@ -53,6 +53,42 @@ the GitHub issue https://github.com/minishift/minishift/issues/179[#179].
 The link:../command-ref/minishift_delete{outfilesuffix}[`minishift delete`] command deletes the OpenShift cluster,
 and also shuts down and deletes the Minishift VM. No data or state are preserved.
 
+[[choosing-iso-image]]
+== Choosing the ISO image
+
+When you start Minishift, it downloads a live ISO image that the hypervisor uses to
+provision the host VM.
+
+The following ISO images are available:
+
+- link:https://github.com/minishift/minishift-b2d-iso/releases[Boot2Docker] (Default). This ISO is
+based on the link:https://github.com/boot2docker/boot2docker[Boot2Docker] ISO, which is a
+lightweight Linux distribution optimized for containerized application development. This ISO image
+is optimized for development but not for production, and is much smaller in size from the CentOS ISO image.
+
+- link:https://github.com/minishift/minishift-centos-iso/releases[CentOS]. This ISO contains
+the link:https://www.centos.org/[CentOS] operating system, which is an enterprise-ready Linux
+distribution. This ISO image provides a production-ready environment.
+
+By default, Minishift uses the **Boot2Docker** image. To choose the CentOS ISO instead,
+you can do one of the following actions:
+
+- Set the `--iso-url` flag when you run `minishift start` and enter the download URL of
+the CentOS ISO in the following format:
++
+----
+$ minishift start --iso-url <Web_location_of_ISO_image>
+----
+- Download the CentSO ISO image from the link:https://github.com/minishift/minishift-centos-iso/releases[releases page]
+and enter the path to the ISO image location in the following format:
++
+----
+$ minishift start --iso-url file:///<path_to_ISO_image>
+----
+
+NOTE: You cannot run Minishift with both ISO images concurrently. To switch between
+ISO images, you must stop and restart Minishift with the ISO image that you want to use.
```

Comment:
_stop and restart_, that won't help. This will just restart the existing VM. You need to delete the Minishift instance. So you need a `minishift delete` followed by a `minishift start` with the correct iso-url.